src/app/views/homepage/homepage.component.html

```diff
@@ -29,7 +29,7 @@ <h1 class="card-header">
       <img src="../../../assets/images/Faculty/PRamamurthy.png" class="img-thumbnail" alt="">
       Chairman's Message</h1>
     <div class="card-body">
-      <p class="card-text d-none d-md-block" style="font-size: medium;;">
+      <p class="card-text d-none d-md-block" style="font-size: medium; text-align: justify;">
```

Comment:
Explain why in all the files you have added text-align : justify
Is it needed?  If yes show image representation of the difference

Reply:
![Screenshot 2024-03-26 171610](https://github.com/ICWaR/icwar.github.io/assets/138218874/09c99ae8-f961-4111-b9af-d98d0fbd5f40)
![Screenshot 2024-03-26 171621](https://github.com/ICWaR/icwar.github.io/assets/138218874/f66f0a43-fa40-419f-99ef-6ba0f14924a1)

Two SS, one without Justify, one after Justify. Without justify is there on icwar.github.io